dhis-2/dhis-api/src/main/java/org/hisp/dhis/analysis/ValidationResultView.java

```diff
@@ -0,0 +1,166 @@
+package org.hisp.dhis.analysis;
+
```

Comment:
This looks like a web domain class.

Move to "dhis-web-api" module, package "org.hisp.dhis.webapi.webdomain".